docs/getting-started/integrations/angular.md

```diff
@@ -205,35 +215,42 @@ Then, import the editor in your Angular component and assign it to a `public` pr
 ```ts
 // app.component.ts
 
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
 import { SlashCommand } from 'ckeditor5-premium-features';
 
-import 'ckeditor5/ckeditor5.css';
-import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
-
 @Component( {
-	selector: 'app-root',
-	templateUrl: './app.component.html',
-	styleUrls: [ './app.component.css' ]
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: [ './app.component.css' ],
+    encapsulation: ViewEncapsulation.None
 } )
 export class AppComponent {
-	title = 'angular';
-
-	public Editor = ClassicEditor;
-	public config = {
-		toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
-		plugins: [
-			Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
-		],
-		licenseKey: '<YOUR_LICENSE_KEY>',
-		mention: {
-			// Mention configuration
-		}
-	}
+    title = 'angular';
+
+    public Editor = ClassicEditor;
+    public config = {
+        toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+        plugins: [
+            Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
+        ],
+        licenseKey: '<YOUR_LICENSE_KEY>',
+        // mention: {
```

Comment:
The empty mention configuration option causes problems with types. I put it in the comment.